src/components/StarRating.ts

```diff
@@ -56,4 +64,10 @@ export class StarRating extends Component<StarRatingProps, {}> {
         // eg fraction 2 or 0.5, rounds off a rate 1.4 to 1.5, 1.2 to 1.0
         return Math.round(props.initialRate * this.fractions) / this.fractions as number;
     }
+
+    private onChange(rate: number) {
+        if (this.props.handleOnChange) {
+            this.props.handleOnChange(Number(rate < 1 ? 1 : rate));
```

Comment:
there's a bug in the ratings widget where its possible to pass 0. yet while rating that values start from 1.